src/test/resources/messages/tierCalculationRequired.json

```diff
@@ -2,7 +2,7 @@
   "Type" : "Notification",
   "MessageId" : "D08B91F6-B2C2-4DA6-8E2A-6F748BF583BD",
   "TopicArn" : "arn:aws:sns:eu-west-2:754256621582:cloud-platform-Digital-Prison-Services-453cac1179377186788c5fcd12525870",
-  "Message" : "{\"offenderId\":2500531599,\"crn\":\"X405099\",\"sourceId\":100692,\"eventDatetime\":\"2020-04-23T152:30:55\"}",
+  "Message" : "{\"offenderId\":2500531599,\"crn\":\"X405099\",\"nomsNumber\":null,\"sourceId\":100692,\"eventDatetime\":\"2020-04-23T152:30:55\"}",
```

Comment:
Thanks for this, the fix is looking good. Just one small thing, this data isn't used in any tests of the text filter so it's not really testing your code. Might be better to make the same change to `externalMovement.json` instead as this is used in test `Should filter on text type`